controller.go

```diff
@@ -40,6 +40,9 @@ type Controller struct {
 	Validation *Validation            // Data validation helpers
 }
 
+// The map of controllers, controllers are mapped by using the namespace|controller_name as the key
```

Comment:
should godocs start with the name of the identifier followed by a description? E.g. "controllers is a list of controllers mapped using..."?